Remove Intercom

rob-baron opened the pull request:
Intercom (https://www.intercom.com) is a third party website that enables communication between users and developers.  It does not seem to be HIPAA, and in any case will violate commonly implemented privacy policies.

rob-baron committed:
RemoveIntercomm

Removing the intercom service as it is a peer to peer networking service
through a third party which potentially violates HIPAA.

rob-baron committed:
Remove Intercom

This removes the intercom (a peer to peer) javascript feature that communicates though a third
party website.  This will be problematic for HIPAA compliance.

lucashxu commented on troposphere/static/js/components/SplashScreen.jsx:
```diff
@@ -37,25 +37,8 @@ export default React.createClass({
-                });
-            }
-
             if (Raven && Raven.isSetup()){
```
We do want to clean this too right?

rob-baron replied:
Raven is removed in a different PR that does not depend on this PR.  I have removed the isEmulatedUser - but removing the emulator really belongs in a separate PR.

rob-baron committed:
RemoveInterComm

Cannot clean up the emulator part as it should be a differen PR.  The emulator
allows the interface to be used by an amistrator to use the identity of a different user
like su command under unix.

rob-baron committed:
Merge branch 'master' into RemoveInterCom

rob-baron committed:
Update SplashScreen.jsx
